logtail/logtail.go

```diff
@@ -431,6 +431,21 @@ func (l *Logger) encodeText(buf []byte, skipClientTime bool) []byte {
 	// For now just factor in a dozen.
 	overhead += 12
 
+	// Put a sanity cap on buf's size.
+	max := 16384
```

Comment:
16 << 10 looks less mysterious

After 4096 I start forgetting my powers of two in decimal

Reply:
I feel the opposite, but sure, done.